.github/workflows/maven.yml

```diff
@@ -70,6 +70,9 @@ jobs:
         elasticsearch-version: 8
         config: |
           http.port: 9200
+          xpack.security.enabled: true
+        env:
```

Comment:
Same here, enviroment variables aren't valid for the github action https://github.com/ankane/setup-elasticsearch/tree/v1